src/main/java/org/betonquest/betonquest/objectives/BlockObjective.java

```diff
@@ -28,42 +30,64 @@ public class BlockObjective extends CountingObjective implements Listener {
 
     private final boolean noSafety;
 
+    private final CompoundLocation location;
+
+    private final boolean hasLocation;
+
+    private final boolean ignorecancel;
+
     public BlockObjective(final Instruction instruction) throws InstructionParseException {
         super(instruction);
         selector = instruction.getBlockSelector();
         exactMatch = instruction.hasArgument("exactMatch");
         targetAmount = instruction.getVarNum();
         noSafety = instruction.hasArgument("noSafety");
+        final String stringlocation = instruction.getOptional("loc", "false");
+        location = "false".equalsIgnoreCase(stringlocation) ? null : new CompoundLocation(instruction.getPackage(), stringlocation);
+        hasLocation = !"false".equalsIgnoreCase(stringlocation);
+        ignorecancel = instruction.hasArgument("ignorecancel");
+
```

Comment:
Maybe not add an extra new line at the end? This happened to all edited methods...
```suggestion
```

Reply:
Tried to remove all i could find and not make the same mistake again.